### ISSUE TITLE
Angular 6 support for new angular poject template.

### ISSUE THREAD

**Issue by hafeefshaik** [aspnet/templating#541]
As of now, when I create a new project, I could see using the older version(5.2.0) of angular. Please provide the support for Angular6.

**Comment by doggy8088** [comment]
In the VSCode's terminal, when running `dotnet build` or `dotnet run` will produce lots of progress info.  Hope this template can turn if off by default.  Here is the sample `package.json` settings:

```
{
  "name": "client-app",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve --no-progress",
    "build": "ng build --no-progress",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
```

**Comment by ryanbrandenburg** [comment]
Closed in #581.

**Comment by hafeefshaik** [comment]
@ryanbrandenburg can you please suggest, how to upgrade existing project(angular template) from angular5 to angluar6.
